packages/webapp/pages/account/profile.tsx

```diff
@@ -34,6 +42,28 @@ const AccountProfilePage = (): ReactElement => {
     e.preventDefault();
     const values = formToJson<UpdateProfileParameters>(formRef.current);
 
+    if (values.username !== user.username) {
```

Comment:
Nice this was the only place it wasn't validating?
(Pretty sure we have it in onboarding/signup flows, but maybe double check the toast)

Reply:
Right. Let me verify that end. If so, it would probably be better to have this inside the calling function to cover all areas.